app/controllers/idv/in_person/address_controller.rb

```diff
@@ -73,7 +95,14 @@ def confirm_in_person_state_id_step_complete
       def confirm_in_person_address_step_needed
         return if pii_from_user && pii_from_user[:same_address_as_id] == 'false' &&
                   !pii_from_user.has_key?(:address1)
-        redirect_to idv_in_person_ssn_url
+        return if request.referer == idv_in_person_verify_info_url
+        redirect_to idv_in_person_verify_info_url
```

Comment:
The referer check looks good (although I've been discovering lately that relying on the referer is not ideal). Why are you redirecting to verify_info instead of ssn at the end?

Reply:
i'm using `confirm_ssn_step_needed` instead to redirect to ssn. Line 99 will redirect us to verify info in case a user changes the url to `in_person_proofing/address` after having completed all the steps.